musig/src/counterparty.rs

```diff
@@ -1,6 +1,5 @@
-use super::multikey::Multikey;
-use super::VerificationKey;
-use crate::errors::VMError;
+use super::errors::MuSigError;
```

Comment:
that's a bit JuMpiNg, let's call it `MusigError`? And use `Musig` instead of `MuSig` everywhere in the source code for that matter?

Reply:
haha SounDs GoOd

Reply:
done!